lib/compat/wordpress-6.8/preload.php

```diff
@@ -13,17 +13,30 @@ function gutenberg_block_editor_preload_paths_6_8( $paths, $context ) {
 	if ( 'core/edit-site' === $context->name || 'core/edit-post' === $context->name ) {
 		$stylesheet       = get_stylesheet();
 		$global_styles_id = WP_Theme_JSON_Resolver_Gutenberg::get_user_global_styles_post_id();
+		$paths[]          = array( '/wp/v2/global-styles/' . $global_styles_id, 'OPTIONS' );
 		$paths[]          = '/wp/v2/global-styles/themes/' . $stylesheet . '?context=view';
 		$paths[]          = '/wp/v2/global-styles/themes/' . $stylesheet . '/variations?context=view';
-		$paths[]          = array( '/wp/v2/global-styles/' . $global_styles_id, 'OPTIONS' );
+
+		// Remove duplicate or unnecessary global styles paths.
 		$excluded_paths[] = '/wp/v2/global-styles/themes/' . $stylesheet;
 		$excluded_paths[] = '/wp/v2/global-styles/' . $global_styles_id;
-	}
-	foreach ( $paths as $key => $path ) {
```

Comment:
If you care about minimising the diff, this can still be left out of the condition?